libyul/optimiser/StructuralSimplifier.cpp

```diff
@@ -137,8 +140,23 @@ void StructuralSimplifier::simplify(std::vector<yul::Statement>& _statements)
 
 				return s;
 			}
-			else
-				return {};
+
+			// Remove cases with empty body if no default case exists
```

Comment:
If you need a comment to document a part of a large block, this usually means you should extract the parts out into functions.

Reply:
I wouldn't consider this block large though ;) But if we do that, we probably should do it with every if-branch above, too.